### ISSUE TITLE
Invalid byte sequence when scanning music

### ISSUE THREAD

**Issue by bwa-** [owncloud/music#338]
Just installed this. Scan fails after some files. Following message in owncloud.log:

{"app":"index","message":"Doctrine\DBAL\DBALException: An exception occurred while executing 'SELECT \"artist\".\"name\", \"artist\".\"image\", \"artist\".\"id\" FROM \"oc_music_artists\" \"artist\" WHERE \"artist\".\"user_id\" = ? AND \"artist\".\"name\" = ?' with params [\"admin\", \"\xff\xfe\"]:\n\nSQLSTATE[22021]: Character not in repertoire: 7 FEHLER:  ung\u00fcltige Byte-Sequenz f\u00fcr Kodierung \u201eUTF8\u201c: 0xff","level":4,"time":"2015-01-14T19:49:57+00:00"}

Music version is 0.3.4, OC is 7.0.4 (stable)
Database is Postgresql (9.3.5)

Refreshing the site seems to trigger the scan again, with the same outcome.

Added info:
### Steps to reproduce
1. Install app
2. Restart OC
3. Open App, wait until server load has dropped. Scan stuck somewhere? Check logs for message above.
### Expected behaviour

Scan should complete, at least skipping unusable titles.
### Actual behaviour

Scan stops at a seemingly random title (have had it stop at different index numbers)
### Server configuration

**Operating system**:
Linux rpi 3.12.35-1-ARCH #1 PREEMPT Tue Dec 23 07:14:51 MST 2014 armv6l GNU/Linux

**Web server:**
nginx version: nginx/1.6.2

**Database:**
postgres (PostgreSQL) 9.3.5

**PHP version:**
PHP 5.6.4 (cli) (built: Dec 19 2014 00:30:13)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2014 Zend Technologies
    with XCache v3.2.1, Copyright (c) 2005-2014, by mOo
    with XCache Cacher v3.2.1, Copyright (c) 2005-2014, by mOo

**ownCloud version:** (see ownCloud admin page)
7.0.4 (stable)

**Updated from an older ownCloud or fresh install:**
Updated

**List of activated apps:**
Activity
Calendar
Contacts
Deleted files
Documents
External storage support
Full Text Search
Pictures
Share Files
Updater
Versions
Video Viewer
Music

**The content of config/config.php:**

```
Insert your config.php content here
<?php
$CONFIG = array (
  'instanceid' => 'ocf360a26a98',
  'passwordsalt' => 'xxxxx',
  'trusted_domains' =>
  array (
    0 => 'rpi',
    1 => 'xxxxx',
  ),
  'datadirectory' => 'xxxxx',
  'dbtype' => 'pgsql',
  'version' => '7.0.4.2',
  'dbname' => 'owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'owncloud',
  'enable_previews' => 'false',
  'dbpassword' => 'xxxxx',
  'installed' => true,
  'theme' => '',
  'maintenance' => false,
  'forcessl' => true,
  'secret' => 'xxxxx',
  'overwrite.cli.url' => 'https://rpi',
);
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
local

**Are you using encryption:** yes/no
no
### Client configuration

**Browser:**
FF/IE

**Operating system:**
Windows 7, Arch Linux

**Comment by paulijar** [comment]
It's likely that this issue no longer happens in the latest Music app version (0.4.4). This looks like some kind of UTF-16 vs UTF-8 encoding mismatch in the scanned file. If this has not got fixed by some earlier change, then at latest #576 should prevent scanning failure on such files.

**Comment by paulijar** [comment]
I assume this has got fixed.
